src/gentropy/dataset/variant_index.py

```diff
@@ -298,3 +298,292 @@ def get_loftee(self: VariantIndex) -> DataFrame:
+         - probably_damaging: 0.908-1: 0.25-1
+         - if assessment is unknown: None
+
+        Args:
+            assessment (Column): PolyPhen assessment.
+            score (Column): PolyPhen score.
+
+        Returns:
+            Column: Normalised PolyPhen score.
+        """
+        return (
+            f.when(assessment == "unknown", f.lit(None).cast(t.DoubleType()))
+            .when(score <= 0.446, cls._rescaleColumnValue(score, 0, 0.446, -1.0, -0.25))
+            .when(
+                score <= 0.908,
+                cls._rescaleColumnValue(score, 0.446, 0.908, -0.25, 0.25),
+            )
+            .when(score > 0.908, cls._rescaleColumnValue(score, 0.908, 1.0, 0.25, 1.0))
+        )
+
+    @classmethod
+    def _normalise_alpha_missense(
+        cls: type[InSilicoPredictorNormaliser],
+        score: Column,
+    ) -> Column:
+        """Normalise AlphaMissense scores.
+
+        Logic: AlphaMissense scores are divided into three categories:
+         - 0-0.06: -1.0--0.25
+         - 0.06-0.77: -0.25-0.25
+         - 0.77-1: 0.25-1
+
+        Args:
+            score (Column): AlphaMissense score.
+
+        Returns:
+            Column: Normalised AlphaMissense score.
+        """
+        return (
+            f.when(score < 0.06, cls._rescaleColumnValue(score, 0, 0.06, -1.0, -0.25))
+            .when(score < 0.77, cls._rescaleColumnValue(score, 0.06, 0.77, -0.25, 0.25))
+            .when(score >= 0.77, cls._rescaleColumnValue(score, 0.77, 1, 0.25, 1))
+        )
+
+    @classmethod
+    def _normalise_splice_ai(
+        cls: type[InSilicoPredictorNormaliser],
+        score: Column,
+    ) -> Column:
+        """Normalise SpliceAI scores.
+
+        Logic: SpliceAI scores are divided into three categories:
+         - 0-0.2: -1.0--0.25
+         - 0.2-0.5: -0.25-0.25
+         - 0.5-1: 0.25-1
+
+        Args:
+            score (Column): SpliceAI score.
+
+        Returns:
+            Column: Normalised SpliceAI score.
+        """
+        return (
+            f.when(score <= 0.2, cls._rescaleColumnValue(score, 0, 0.2, -1, -0.25))
+            .when(score <= 0.5, cls._rescaleColumnValue(score, 0.2, 0.5, -0.25, 0.25))
+            .when(score > 0.5, cls._rescaleColumnValue(score, 0.5, 1, 0.25, 1))
+        )
+
+    @classmethod
+    def _normalise_pangolin(
+        cls: type[InSilicoPredictorNormaliser],
+        score: Column,
+    ) -> Column:
+        """Normalise Pangolin scores.
+
+        Logic: Pangolin scores are divided into two categories:
+            - 0-0.14: 0-0.25
+            - 0.14-1: 0.75-1
+
+        Args:
+            score (Column): Pangolin score.
+
+        Returns:
+            Column: Normalised Pangolin score.
+        """
+        return f.when(
+            f.abs(score) > 0.14, cls._rescaleColumnValue(f.abs(score), 0.14, 1, 0.0, 1)
+        ).when(
+            f.abs(score) <= 0.14,
+            cls._rescaleColumnValue(f.abs(score), 0, 0.14, -1, 0.0),
```

Comment:
The description and the implementation don't agree. I think the description is better. Either that or: 0-0.5 and 0.5-1. But I wouldn't make it negative as we are not really sure we are talking about benign variation